modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,47 @@
+.Procedure
+
+. When using a JetBrains IDE requesting a registration, see xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[].
+
+. Add the following component to the workspace devfile:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: "__<URL_to_meta.yaml>__" <1>
+----
++
+<1> __<URL_to_meta.yaml>__: HTTPS resource defining the editor metadata. It can be:
++
+* IntelliJ IDEA Community prebuilt IntelliJ-based IDEs image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml
```

Comment:
* Still failing when using https://workspaces.openshift.com/#https://github.com/themr0c/che-docs/tree/test-intellij

* Failing when modifying a Java Gradle stack on https://workspaces.openshift.com

```
Successfully pulled image "quay.io/che-incubator/che-idea:latest" in 282.196626ms

Created container che-ideaif5

Started container che-ideaif5

Pulling image "registry.redhat.io/codeready-workspaces/plugin-java11-rhel8@sha256:f90e09397caa4dd5cd3476e495adfc0096272c0ba746a6e32215175c541ae058"

Failed to run the workspace: "The following containers have terminated: vscode-javay1h: reason = 'Completed', exit code = 0, message = 'null'"
```

Reply:
* Success on a blank devfile.!!!

@vzhukovs maybe we need to state what needs to be absent/removed from a devfile?

Reply:
Added a step:

. Remove the plugins or commands defined for the Theia editor from the workspace devfile.

Removed this statement, as the workspace is failing when a theia plugin is still present in the devfile.

The workspace running with this editor ignores the plugins or commands defined for Theia